package.json

```diff
@@ -12,7 +12,6 @@
     "coveralls": "nyc report --reporter=text-lcov | coveralls",
     "test": "export NODE_ENV=test && npm run undomigration && npm run migration && npm run seeds && nyc --reporter=html --reporter=text mocha ./test --no-timeout --exit --require @babel/register",
     "dev": "npm run undoseeds && npm run seeds && nodemon --exec babel-node ./src/app.js"
```

Comment:
@patric is this "npm  dev" working as expected?

Reply:
No. I did not want to change it before informing the team

Reply:
@minega25 it's missing the ```npm run migration``` after undoseeds.